test/WebJobs.Script.Tests/Description/FunctionDescriptorProviderTests.cs

```diff
@@ -203,54 +203,6 @@ public async Task CreateTriggerParameter_WithNoBindingMatch_ThrowsExpectedExcept
             Assert.Contains("someInvalidTrigger", ex.Message);
         }
 
-        [Theory]
```

Comment:
This was just testing a wrapper that directly called `Utility.ValidateBinding(binding)` using the same test cases.

Reply:
I think these tests should remain. They're testing the virtual FunctionDescriptorProvider.ValidateBinding method impl which right now is basically just Utility.ValidateBinding, but it does also have some additional logic. And in the future more might be added that we need to test.

Reply:
Yes, there's a test for the error as well. Will revert this then.